src/Scripting/CSharpTest/InteractiveSessionTests.cs

```diff
@@ -466,6 +466,47 @@ public void CompilationChain_GlobalImports()
             Assert.Equal(Environment.ProcessorCount, state.Result.ReturnValue);
         }
 
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/27356")]
+        public void CompilationChain_Accessibility()
```

Comment:
>CompilationChain_Accessibility [](start = 20, length = 30)

Is it possible to add similar test for VB?

Reply:
Not an analogous test because the VB scripting implementation is not resilient to errors (it throws an exception when a script with some kinds of errors is created, and it is therefore not possible to get back the resulting compilation).

I will test a subset of the scenarios in VB. The VB scripting implementation is too incomplete to reliably test though.

---
In reply to: [192576081](https://github.com/dotnet/roslyn/pull/27312#discussion_r192576081) [](ancestors = 192576081)